Szymon/Zadanie7/src/test/java/Szymon7/page/accountServicesPage/AccountServicesPage.java

```diff
@@ -0,0 +1,42 @@
+package Szymon7.page.accountServicesPage;
+
+import Szymon7.page.common.Page;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AccountServicesPage extends Page {
+    @FindBy(xpath = "//a[contains(text(),'Transfer Funds')]")
+    private WebElement findTransferFunds;
+    @FindBy (xpath = "//input[@value='Transfer']")
+    private WebElement transferButton;
+    @FindBy (xpath = "//a[contains(text(),'Accounts Overview')]")
+    private WebElement accountsOverviewLink;
+    @FindBy (xpath = "//select[@id='fromAccountId']/*")
+    private WebElement selection;
+
+    public AccountServicesPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public TransferFundsPage clickTransferFundsLink() {
+        findTransferFunds.click();
+        if (selection.getText().equals("UNDEFINED"))
```

Comment:
To jest obejscie jakiegos buga na stronie?

Reply:
dokładnie tak, nie wiem dlaczego ale czasami po za szybkim zrobieniu usera i wejsciu na stronę ustawiało jako undefined dropdown do przelewu (bez innej opcji do wyboru), ponowne kliknięcie już ok